### PR TITLE
removed wildfly profile for optaplanner

### PR DESCRIPTION
**Pull request by mbiarnes**
wildfly profile was removed in optaplanner PRs and the the the deploy job of the repository

### CODE REVIEW

**Comment by mbiarnes** [comment]
@mareknovotny tested locally - it works!

File: job-dsls/jobs/deploy_jobs.groovy [hunk 1]
```diff
@@ -70,7 +70,13 @@ def final REPO_CONFIGS = [
         ],
         "optaplanner"               : [
                 ircNotificationChannels: ["#optaplanner-dev"],
-                downstreamRepos        : ["optaplanner-wb", "optaweb-employee-rostering"]
+                downstreamRepos        : ["optaplanner-wb", "optaweb-employee-rostering"],
```
Comment (mareknovotny) [inline job-dsls/jobs/deploy_jobs.groovy]: I think there is also optaweb-vehicle-rooting downstream repository

Comment (mbiarnes) [reply job-dsls/jobs/deploy_jobs.groovy]: OK  -I will change this

Comment (mbiarnes) [reply job-dsls/jobs/deploy_jobs.groovy]: @mareknovotny DONE

**Comment by mareknovotny** [comment]
@ge0ffrey are you fine with the downstream repos config now?

**Comment by ge0ffrey** [comment]
Yes, +1 - thanks!

**Pull request merged**